docs/06-concepts/18-webserver/_category_.json

```
@@ -0,0 +1,4 @@
+{
+  "label": "Web Server",
```

Suggestion: Only use large caps for first word. please fix this for all sub pages as well.

```suggestion
  "label": "Web server",
```